src/wrapped/wrappedsdl2.c

```diff
@@ -710,8 +872,16 @@ EXPORT void* my2_SDL_GL_GetProcAddress(x86emu_t* emu, void* name)
         strcpy(tmp, "my_");
         strcat(tmp, rname);
         symbol = dlsym(emu->context->box86lib, tmp);
-    } else 
-        symbol = my->SDL_GL_GetProcAddress(name);
+    } else {
+        char buf[200] = {};
+            symbol = my_glXGetProcAddress(emu, name);
```

Comment:
what is the point of this? both function will do the same mapping, and the libGL is supposed to be the same.

Reply:
That's true, this workaround was a tiny bit too jerry rigged.

Reply:
The correct way would've been having used `SDL_VIDEO_GL_DRIVER` to point towards the shim on egl-based backends, I was using `SDL_OPENGL_LIBRARY` instead and thus got the weird behavior. I'll be reverting these.